js/app.js

```diff
@@ -1 +1,179 @@
-console.log('Hello frontend');
+// isGameWon ends the game and ability to click the page when isGameWon results in true
+let isGameWon = false;
+
+// attaching js variables to the front end
+let zeroZero = document.getElementById("zero-zero");
+let oneZero = document.getElementById("one-zero");
+let twoZero = document.getElementById("two-zero");
+let zeroOne = document.getElementById("zero-one");
+let oneOne = document.getElementById("one-one");
+let twoOne = document.getElementById("two-one");
+let zeroTwo = document.getElementById("zero-two");
+let oneTwo = document.getElementById("one-two");
+let twoTwo = document.getElementById("two-two");
+
+// clickable function
+zeroZero.addEventListener("click", playValue(0, 0));
+oneZero.addEventListener("click", playValue(1, 0));
+twoZero.addEventListener("click", playValue(2, 0));
+zeroOne.addEventListener("click", playValue(0, 1));
+oneOne.addEventListener("click", playValue(1, 1));
+twoOne.addEventListener("click", playValue(2, 1));
+zeroTwo.addEventListener("click", playValue(0, 2));
+oneTwo.addEventListener("click", playValue(1, 2));
+twoTwo.addEventListener("click", playValue(2, 2));
+
```

Comment:
This was pretty cool to see and talk about earlier in class today.  I wanted to note that you are utilizing Closure here because the playValue functions are returning functions which are utilizing the x and y values called as arguments in the playValue functions when you call them here on lines 16-24.  Closure often makes sense without having to think about it, but is a good thing to understand when it comes to interviews.  I would encourage you to read up more on them when you get a little free time.  I struggled for months to fully understand closures, but I may be able to point in you a direction if you are struggling with any parts of it.